### PR TITLE
qa: fix manager module paths

### PR DESCRIPTION
**Pull request by noahdesu**

### CODE REVIEW

File: qa/run-standalone.sh [hunk 1]
```diff
@@ -12,8 +12,6 @@ function get_cmake_variable() {
 }
 
 function get_python_path() {
-    local ceph_lib=$1
```
Comment (noahdesu) [inline qa/run-standalone.sh]: looks fishy! but it appears no callers pass in a $1 arg.

**Comment by dzafman** [comment]
@noahdesu It would be nice if you could pull the following commit into this pull request: https://github.com/dzafman/ceph/commit/046a33c08c30e7d5c02866e69792d04c9631166b

**Comment by dzafman** [comment]
TESTS PASSED (standalone only)
http://pulpito.ceph.com/dzafman-2018-08-17_17:03:49-rados-qa-mgr-mod-path-distro-basic-smithi

**Comment by noahdesu** [comment]
Added dzafman@046a33c

**Comment by dzafman** [comment]
We need to re-run tests.

My change will result in the following difference in the log and will still pass the test

```
2018-08-18T00:22:20.496 ...ceph-helpers.sh:1931: test_flush_pg_stats:  test 2995
```

```
2018-08-18T00:22:20.496 ...ceph-helpers.sh:1931: test_flush_pg_stats:  test 2995 -gt 0
```

**Comment by dzafman** [comment]
Testing passed:
http://pulpito.ceph.com/dzafman-2018-08-17_17:03:49-rados-qa-mgr-mod-path-distro-basic-smithi/

**Comment by dzafman** [comment]
retest check please

**Comment by dzafman** [comment]
retest this please

**Comment by dzafman** [comment]
@noahdesu You should review my commit and then merge it with both of us as reviewers.

**Comment by tchaikov** [comment]
@noahdesu next time, when you merge a PR, could you add one or more `Reviewed-by` line in the commit message of the merge ? so we can track back to the reviewer(s) without looking it up on github? it helps when we need to find a reviewer for a change/backport.

**Comment by noahdesu** [comment]
@tchaikov yes of course thanks for pointing this out to me.

**Pull request merged**